chatcommands.py

```diff
@@ -1752,6 +1752,16 @@ def allspam(msg, url, alias_used="allspam"):
 
 def report_post(url, reported_by, reported_in=None, blacklist_by=None, operation="report", custom_reason=None):
     """ Scan or report a single post """
+    # Arguments:
+    # - url: url to the post being reported
+    # - reported_by: user reporting the post
```

Comment:
What is `reported_by`? A string? A dict? Similar information should be provided for the other arguments too.